modular_skyrat/modules/colony_fabricator/code/machines/solid_fuel_generator.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/power/port_gen/pacman/solid_fuel
-	name = "solid fuel generator"
-	desc = "The second most common generator design in the galaxy, second only to the PACMAN. \
+	name = "S.O.F.I.E. solid fuel generator"
```

Comment:
Might want to add what "S.O.F.I.E." stands for in the description, too. Also, here, so it's more in-line with how the P.A.C.M.A.N.s are named like.
```suggestion
	name = "\improper S.O.F.I.E.-type portable generator"
```